app/views/UserNotificationPreferencesView/index.tsx

```diff
@@ -51,43 +65,43 @@ class UserNotificationPreferencesView extends React.Component {
 		this.setState({ preferences, loading: true });
 	}
 
-	findDefaultOption = key => {
+	findDefaultOption = (key: TKey) => {
 		const { preferences } = this.state;
 		const option = preferences[key] ? OPTIONS[key].find(item => item.value === preferences[key]) : OPTIONS[key][0];
 		return option;
 	};
 
-	renderPickerOption = key => {
+	renderPickerOption = (key: TKey) => {
 		const { theme } = this.props;
 		const text = this.findDefaultOption(key);
-		return (
-			<Text style={[styles.pickerText, { color: themes[theme].actionTintColor }]}>
-				{I18n.t(text?.label, { defaultValue: text?.label, second: text?.second })}
-			</Text>
-		);
+		return <Text style={[styles.pickerText, { color: themes[theme].actionTintColor }]}>{I18n.t(text?.label)}</Text>;
```

Comment:
Because here there aren't `text?.label` or `text?.second`, add to this, none of the TKey receives defaultValue os second when we search inside the i18n